apcd-cms/src/apps/utils/apcd_database.py

```diff
@@ -766,20 +773,14 @@ def create_threshold_exception(form, sub_data):
             status
         ) VALUES (%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s)
         """
-        values = (
-            form["business-name"],
-            sub_data[1],
-            sub_data[2],
-            sub_data[3],
-            _clean_value(form['requestor-name']),
-            _clean_email(form['requestor-email']),
-            "threshold",
-            _clean_date(form['expiration-date']),
-            _clean_value(form['file_type']),
-            _clean_value(form['field-threshold-exception']),
-            _clean_value(form['threshold-requested']),
-            _clean_value(form['justification']),
-            "pending"
+
+        conn = psycopg2.connect(
+            host=APCD_DB['host'],
+            dbname=APCD_DB['database'],
+            user=APCD_DB['user'],
+            password=APCD_DB['password'],
+            port=APCD_DB['port'],
+            sslmode='require'
```

Comment:
Looks like more diff is being generated from this var being moved within this function - could it be moved back to lessen diff?